agent/agentexec/cli_linux_test.go

```diff
@@ -171,3 +202,14 @@ func execArgs(oom int, nice int) []string {
 	execArgs = append(execArgs, "--")
 	return execArgs
 }
+
+func setCaps(t *testing.T, bin string, caps ...string) error {
+	t.Helper()
+
+	setcap := fmt.Sprintf("sudo setcap %s=ep %s", strings.Join(caps, ", "), bin)
+	out, err := exec.CommandContext(context.Background(), "sh", "-c", setcap).CombinedOutput()
```

Comment:
Pretty sure sudo should exit non-zero in case it would prompt for a password, just checking if this is indeed the case and the test won't freeze up if we don't have passwordless sudo.

Reply:
I thought it would too if you didn't hook up `stdin` but that doesn't seem to be the case. Fixed now.